assets/css/homepage.scss

```diff
@@ -0,0 +1,233 @@
+---
+---
+
+// Imports -------------- //
+
+@import 'lib/bourbon/bourbon';
+@import 'lib/neat/neat';
+@import 'all';
+
+// Variables ------------ //
+
+$text-width:      660px;
+$section-padding: 7.2rem;
+$font-lead:   2rem;
+
+// Styles --------------- //
+
+p {
+  font-family: $font-sans;
+  font-size: 1.7rem;
+}
+
+.skipnav {
+  padding: 10px 15px;
+  position: absolute;
+  top: -42px;
+  left: 0px;
+  color: $color-base;
+  background: transparent;
+  @include transition(top 1s ease-out, background 1s linear);
+  z-index: 100;
+  &:focus {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    background: white;
+    outline: 0;
+    @include transition(top .1s ease-in, background .5s linear);
+  }
+}
+
+.usa-disclaimer {
+  background-color: $color-gray-lighter;
+  font-size: 1.5rem;
+  padding: {
+    bottom: .5rem;
+    top: .5rem;
+  }
+}
+
+.usa-disclaimer-stage {
+  float: right;
+}
+
+.usa-intro {
+  max-width: $text-width;
+}
+
+.usa-section-examples {
+  @include media($medium-screen) {
+    .usa-width-five-sixths {
+      @include shift(1);
+    }
+  }
+}
+
+.usa-section {
+  padding: {
+    bottom: $section-padding;
+    top: $section-padding;
+  }
+
+  h2 {
+    font-size: 3rem;
+  }
+}
+
+.usa-button-block {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.usa-cta {
+  clear: both;
+  display: block;
+}
+
+.usa-banner {
+  @include media ($medium-screen) {
+    background-image: url("../img/home/homepage_illustrations_flag_cropped.png");
+    background-position: right bottom;
+    background-repeat: no-repeat;
+  }
+  background-color: $color-primary-darkest;
+  background-image: none;
+  color: $color-white;
+  padding: {
+    bottom: $section-padding;
+    top: 2rem;
+  }
+
+  .usa-lead {
+    color: $color-white;
+    margin-bottom: 1rem;
+  }
+
+  h1 {
+    color: $color-white;
+
+    @media (min-width: $medium-screen) {
+      font-size: 5.6rem;
+      padding-top: 4.5rem;
+    }
+  }
+
+  .usa-text-small {
+    font-size: 1.3rem;
+    margin-top: 0.1em;
+    text-align: center;
+  }
+
+  .usa-button-outline-inverse {
+    margin-right: 0;
+  }
+}
+
+.usa-banner-link-top {
+  color: $color-white;
+  float: right;
+  font-weight: $font-bold;
+
+  &:visited {
+    color: $color-white;
+  }
+}
+
+.usa-banner-content {
+  max-width: 48rem;
```

Comment:
This is the max-width needed to accomplish the design of the main banner text content. We are not using the grids bc we do not want it to flex and change the text per line as you update the width of the site. It needs to be the same width until it drops to a very small size.